docs-src/stargate-develop/modules/develop/pages/tooling.adoc

```diff
@@ -161,35 +161,36 @@ such as `createTable`, or `dropTable`.
 * `/graphql/<keyspace>`
 ** An API for querying and modifying data in your tables using GraphQL fields.
 
-== Swagger resources
+== REST and Document API resources
 
 The Swagger UI allows you to interact with the REST and Document APIs in a web browser.
  
 === Using the Swagger UI
 
 ifeval::["{product}" == "Astra DB"]
-In Astra, go to the Connect tab for your database, choose GraphQL under the
-`Connect using an API` and you'll see instructions for accessing the playground.
-In a browser, you can connect at `{base_rest_url}/api/rest/swagger-ui/`
+In Astra, go to the Connect tab for your database, choose REST API or Document API under the
+`Connect using an API` and you'll see instructions for accessing the Swagger UI.
+In a browser, you can connect at `{base_rest_url}/api/rest/swagger-ui/`.
```

Comment:
One additional note - we are breaking out the swagger for docs API to it's own path, it will be `{base_rest_url}/api/docs/swagger-ui/`, but this is not live yet. So we will need another PR once that change goes live.

Reply:
Thanks for the heads-up.